net/gnunet/Makefile

```diff
@@ -72,6 +72,10 @@ define Package/gnunet/description
  well as their helpers.
 endef
 
+define Package/gnunet/config
+	source "$(SOURCE)/Config.in"
```

Comment:
Why not have those 3 lines directly in the Makefile instead of sourcing and creating an additional file?